lib/model/job.js

```diff
@@ -21,6 +21,7 @@ var Job = new mongoose.Schema({
   id: { type: String, default: '' },
   compendium_id: { type: String, default: '' },
   user: { type: String, default: '' },
+  journal: {type: String, default: ''},
```

Comment:
Why does the job need the journal - can the journal not be stored with the compendium for which the job is running?